### ISSUE TITLE
Adjusting days on rolling week widget not working

### ISSUE THREAD

**Issue by Mayfieldtechnologies** [campai-bv/autotask-dashboards-os#64]
Per a comment I found on the main website, I adjusted line 54 in the bootstrap.php file to read:

```
Configure::write('Widget.RollingWeek.daysOfHistory', 14);
```

However, the RollingWeek widget still only shows 7 days of history.  Is there another value somewhere I need to change to get this to show the full 14 days?

Thank you!

**Comment by coencoppens** [comment]
Well, that should do the trick. Maybe there's a cached view file in the way?

**Comment by Mayfieldtechnologies** [comment]
I don't believe there is.  Even when creating a new dashboard it is only showing 7 days of history.  Where do you believe I could look to see if there is a cached view file?

Thanks!
